README.md

```diff
@@ -41,28 +37,24 @@ This project includes configuration and tooling that conforms to Crema's baselin
   ]
 }
 ```
-7. [Install Docker Desktop](https://www.docker.com/products/docker-desktop)
 
 ## 👟 Run
 
 Run the following scripts with `npm run <SCRIPT_HERE>`:
 
-- `start` - start app
-- `new:component` - generate a new component
-- `new:util` - generate a new util(ity)
-- `test:analyze` - run bundle analyzer
-- `test:deps` - run dependency validation tests
-- `test:e2e` - run end-to-end tests
-- `test:lint:fix` - run linter and fix if possible
+- `start` - start the expo bundler
+- `start:ios` - start the expo bundler and an iOS simulator
+- `start:android` - start the expo bundler and an android emulator
+- `start:web` - start the expo bundler and the web app
+- `start:playground` - start component playground with storybook
```

Comment:
```suggestion
- `start:visual` - start component playground with storybook
```
🤷‍♂

Reply:
@roblafeve This command doesn't start the visual tests, it starts the app and opens up the Storybook UI.

Reply:
Right, I was wondering if having a `start:visual` that corresponds with `test:visual` would make sense. It kind of conveys a relationship between them. Something I've wondered about on the web side too. Food for 💭 .

Reply:
Hmmm, that makes sense actually. 🤔

Reply:
🤷‍♂